pkg/workload/workload.go

```diff
@@ -288,3 +341,30 @@ func GetQueueOrderTimestamp(w *kueue.Workload) *metav1.Time {
 func IsAdmitted(w *kueue.Workload) bool {
 	return apimeta.IsStatusConditionTrue(w.Status.Conditions, kueue.WorkloadAdmitted)
 }
+
+// UpdateReclaimablePods updates the ReclaimablePods list for the workload wit SSA.
+func UpdateReclaimablePods(ctx context.Context, c client.Client, w *kueue.Workload, reclaimablePods []kueue.ReclaimablePod) error {
```

Comment:
```suggestion
func ApplyReclaimablePods(ctx context.Context, c client.Client, w *kueue.Workload, reclaimablePods []kueue.ReclaimablePod) error {
```

Reply:
Let's keep "Apply" , for when we only push parts of a workload to the api server.

Reply:
Which this does?

Reply:
`reclaimablePods []kueue.ReclaimablePod ` is not part of the  ` w *kueue.Workload` is used to identify the target.

Reply:
I want to be able to quickly identify which function is using Update (PUT), versus SSA (PATCH). For that reason, I find `Apply` very obvious.

But we can clean this up later.